iDiff/cmd/iDiff_test.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"testing"
+)
+
+type testpair struct {
+	values   []string
+	expected bool
+}
+
+var tests = []testpair{
+	{[]string{}, false},
+	{[]string{"one"}, false},
+	{[]string{"one", "two"}, true},
+	{[]string{"one", "two", "three"}, true},
+	{[]string{"one", "two", "three", "four"}, false},
+}
+
+func TestArgNum(t *testing.T) {
+	for _, test := range tests {
+		valid, err := checkArgNum(test.values)
+		if valid != test.expected {
+			if test.expected {
+				t.Errorf("Expected error but got none")
```

Comment:
I'm a little confused by this. Does expected mean that the test is expected to fail? In that case, wouldn't you still need one more check here: `if err == nil`?